### PR TITLE
Create new-ref-doc-JDN.md

### PR DESCRIPTION
**Pull request by nemethconsulting**
new ref doc written for the to-do svc

### CODE REVIEW

File: docs/api/new-ref-doc-JDN.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## URL
+
+```shell
+{base_url}/tasks/{id}
+```
+
+## Method
+`PATCH`
+
+## Params
```
Comment (rbwatson) [inline docs/api/new-ref-doc-JDN.md:18]: This isn't quite correct.
- `id` is a URL parameter (as shown in the *URL* section)
- `due_date` is a request body property (and should be in the *Request body* section below.

File: docs/api/new-ref-doc-JDN.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+| Header name    | Type   | Description                           |
+| -------------- | ------ | ------------------------------------- |
+| `Content-Type` | string | Must be set to `application/json`     |
+
+## Request body (using cURL)
+
+In this example, we are adjusting the due date for task 5 in the resource
+
+```bash
+curl -X PATCH "{base_url}/tasks/{id}" \
```
Comment (rbwatson) [inline docs/api/new-ref-doc-JDN.md:36]:
```suggestion
curl -X PATCH "{base_url}/tasks/5" \
```
If your example is to update task 5, then the specific `id` should be shown.

File: docs/api/new-ref-doc-JDN.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+| `due_date`     | string | The date in ISO 8601 format         |
+
+## Request headers
+
+| Header name    | Type   | Description                           |
+| -------------- | ------ | ------------------------------------- |
+| `Content-Type` | string | Must be set to `application/json`     |
+
+## Request body (using cURL)
+
```
Comment (rbwatson) [inline docs/api/new-ref-doc-JDN.md:32]: It seems like the request body property from earlier should appear here and the following example should be labeled as an example.

File: docs/api.md [hunk 4]
```diff
@@ -22,4 +22,5 @@ On test and development installations, the server's base address is typically `h
     * [Get task by ID - Doug](api/tasks-get-by-task-id_dmanis.md)
     * [Get tasks by User ID - David](api/tasks-get-tasks-by-user_id_david.md)
     * [Get tasks by User ID - Wendy](api/tasks-get-tasks-by-user_id_WUF.md)
+    * [Update task due date - Jason](api/new-ref-doc-JDN.md)
```
Comment (rbwatson) [inline docs/api.md:25]: added to provide a link to the new topic

File: docs/api/new-ref-doc-JDN.md
Comment (rbwatson) [inline docs/api/new-ref-doc-JDN.md:1]: I moved the file from the documentation root to the `api` directory so it's with all the other reference topics.

**Review by rbwatson — CHANGES_REQUESTED** [review]
I made some changes so that the PR can merge correctly into the docs, but there are couple of minor revisions to make before the topic is ready to merge.

File: docs/api.md (hunk 4 above)
Comment (nemethconsulting) [reply docs/api.md:25]: Thanks.

File: docs/api/new-ref-doc-JDN.md
Comment (nemethconsulting) [reply docs/api/new-ref-doc-JDN.md:1]: Thanks.

File: docs/api/new-ref-doc-JDN.md (hunk 3 above)
Comment (nemethconsulting) [reply docs/api/new-ref-doc-JDN.md:32]: Okay. Did that.

File: docs/api/new-ref-doc-JDN.md (hunk 1 above)
Comment (nemethconsulting) [reply docs/api/new-ref-doc-JDN.md:18]: Moved 'due-date' to Request Body section.

**Review by rbwatson — APPROVED** [review]
LGTM

**Pull request merged**